flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroToRowDataConverters.java

```diff
@@ -101,7 +106,7 @@ private static AvroToRowDataConverter createNullableConverter(LogicalType type)
 	/**
 	 * Creates a runtime converter which assuming input object is not null.
 	 */
-	private static AvroToRowDataConverter createConverter(LogicalType type) {
+	private static AvroToRowDataConverter createConverter(LogicalType type, @Nullable JodaConverter jodaConverter) {
```

Comment:
We can simply access JodaConverter where needed as a singleton? I would not pollute this methods.